### DISCUSSION TITLE
Connect to Lbb as a Webserver on local Network?

### DISCUSSION THREAD

**Discussion by ruebyi** [outdoorbits/little-backup-box#318] [Q&A] [answered]
Hi,
I remember there was a way to connect to lbb via smb.
Right now I‘m trying to get a connection a Access via the iPad Files app, but no connection is established.
Can someone help me what I‘m missing? 
the wiki doesn‘t hav any information about that type of connection. 

Thank you!

**Comment by outdoorbits** [comment, marked as answer]
Just navigate your file browser to smb://10.41.0.1 or what ever your IP 
is. To access the folders you will need your password if set. User is 
'lbb'.

**Reply by ruebyi** [reply]
Just to add my experience:
It took a long time to connect to smb on a pi Zero 2 - sometimes it wouldn't even connect

It's perfect on a pi 5
